mintpy/utils/readfile.py

```diff
@@ -772,6 +772,10 @@ def read_attribute(fname, datasetName=None, metafile_ext=None):
             k = list(set(g1_list) & set(py2_mintpy_stack_files))[0]
         elif 'FILE_TYPE' in atr:
             k = atr['FILE_TYPE']
+            # catech exception for horizontal and vertical decomposition
+            if (k == 'None') and ('SenD' in d1_list or 'horizontal' \
+                in d1_list or 'vertical' in d1_list):
+                k = 'SenD'
```

Comment:
I guess you add this because of the improper FILE_TYPE in the `asc_desc2horz_vert.py` output? If so, we should update there instead, e.g. from 

https://github.com/insarlab/MintPy/blob/6c7ad9ec07462022f021886b92f02164bf842ef6/mintpy/asc_desc2horz_vert.py#L267-L268

to 

```python
if dsname and atr['FILE_TYPE'] in ['ifgramStack', 'timeseries', 'HDFEOS']:
        atr['FILE_TYPE'] = 'displacement'
```

Then no change is needed here in `readfile.py`.

Reply:
Yeah I agree, this would be much cleaner and the more generic designation (i.e. displacement vs. SenD) makes sense to me